editor/src/components/inspector/inspector-strategies/fixed-size-basic-strategy.ts

```diff
@@ -43,7 +36,7 @@ export const fixedSizeBasicStrategy = (
           setExplicitCssValue(value),
           parentFlexDirection,
         ),
-        pushIntendedBoundsAndUpdateGroups([{ target: path, frame: predictedElementSize }]),
+        queueGroupTrueUp(path),
```

Comment:
basically this line is the interesting change in the PR

Reply:
Don't sell your work short, there are many interesting changes in this PR :)